src/componenets/AboutUs/TeamMemberCards.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { MDBCard, MDBCardBody, MDBIcon } from 'mdbreact';
+import PropTypes from 'prop-types';
+function TeamMemberCards({ member }) {
+  const { pic, fullName, specailty, bio } = member;
+  return (
+    <MDBCard
+      className="animate transition duration-500 ease-in-out  
+                        transform hover:-translate-y-1 hover:scale-110 content-start max-w-xs m-5 bg-blue-100 "
```

Comment:
Delete this extra space
```suggestion
                        transform hover:-translate-y-1 hover:scale-110 content-start max-w-xs m-5 bg-blue-100"
```